src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -57,6 +57,14 @@ public static MeterProviderBuilder AddView(this MeterProviderBuilder meterProvid
                 throw new ArgumentException($"Custom view name {name} is invalid.", nameof(name));
             }
 
+            if (instrumentName.IndexOf('*') != -1)
+            {
+                throw new ArgumentException(
```

Comment:
@alanwest Any better wording suggestions?

Reply:
maybe something like...

"Instrument selection criteria is invalid. Instrument name '{instrumentName}' contains a wildcard character. This is not allowed when using a view to rename a metric stream as it would lead to conflicting metric stream names."